test.py

```diff
@@ -633,7 +633,7 @@ def write_junit_failure_report(self, xml_res: ET.Element) -> None:
 class UnitTest(Test):
     standard_args = shlex.split("--overprovisioned --unsafe-bypass-fsync 1 "
                                 "--kernel-page-cache 1 "
-                                "--blocked-reactor-notify-ms 2000000 --collectd 0 "
+                                "--blocked-reactor-notify-ms 2000000 "
                                 "--max-networking-io-control-blocks=100 ")
```

Comment:
This is important, it tells Seastar not to blast collectd packets everywhere.

Reply:
I had hoped I'm removing collectd invocation everywhere with this change. Does that mean I'm not, as Seastar will still be running collectd by default? Should I send a patch over there to change the default to 0 then first?

Reply:
I thought this -  https://github.com/scylladb/seastar/blame/b8582345a10243c0d0cd028a7cc6db6124950ae6/src/core/scollectd.cc#L563 means it's off by default.

Reply:
Better to check

Reply:
Snooping on the traffic (on the multicast address) shows nothing - used a Docker container - unsure if it passed multicast or not (but used host networking, so it was supposed to).
In any case, we can drop this specific change - it's just a change in unit test invocation.

Reply:
was it dropped?

Reply:
Dropped it and pushed.